configure.ac

```diff
@@ -1586,10 +1586,11 @@ dnl ##########################################################################
 dnl ------------------
 dnl check C-Ares library
 dnl ------------------
+c_ares_found=false
```

Comment:
I suppose we should do the actual cleanup and remove the usage of `c_ares_found` and `CARES` in autoconf/automake files? Or do you want to keep this dead code for some reason?

Reply:
I was aware that this is dead now.  But I was actually a bit confused by how we got to this state and the person to ask is on vacation this week.  I was hoping to get the freebsd people up and running again in the meantime.

Reply:
Then probably may be better to merge https://github.com/FRRouting/frr/pull/9404 instead? It fixed the compilation issue without making libc-ares hard dependency.
And later if we decide that it's fine to make it a hard dependency then we can finish the cleanup?

Reply:
that works for me then

Reply:
I'll remove this commit. Can you work with the other PR to get it in?

Reply:
It's merged now.